indico/modules/events/notes/blueprint.py

```diff
@@ -23,5 +24,7 @@
                          defaults={'object_type': object_type})
```

Comment:
The old note editing endpoints are no longer used, right? So you could already remove those.. (endpoint, RH, wtform)